### DISCUSSION TITLE
Bedeutung der LED auf der CCU3

### DISCUSSION THREAD

**Discussion by thoxxxx** [OpenCCU/OpenCCU#2488] [🇩🇪 Generelle Diskussionen]
Hallo,

ich bin vor einigen Wochen von der OriginalFW auf RaspberryMatic umgestiegen. Alles funktioniert soweit ganz gut.

Aber die LED blinkt ständig Lila / Gelb abwechselnd. Es gibt keine Alarm- bzw. Servicemeldungen.

Was bedeutet das? 

Gibt es auch eine Beschreibung, was die LED Meldungen bedeuten. In Wiki hat ich adhoc nichts gefunden.

Danke und Grüße
Thomas

**Comment by ghost** [comment]
Schreiben Sie mehr darüber, was gerade passiert, und senden Sie vielleicht ein paar Screenshots und Fotos. Vielleicht kann dann jemand helfen, da die Informationen derzeit nicht ausreichen.

**Comment by Sineos** [comment]
https://github.com/search?q=repo%3Ajens-maus%2FRaspberryMatic+led&type=wikis

**Comment by thoxxxx** [comment]
Ich habe von ein paar Wochen auf RaspberryMatic umgestellt. Heute habe ich auch das Reset durchgeführt (Wege Speicher).

Nach dem Neustart hast das blau/gelb Blinken aufgehört. Jetzt durchgehend blau. Vielleicht lag es daran?

Grüße
Thomas

**Comment by thoxxxx** [comment]
So jetzt habe ich wieder das Phänomen:

Die große LED blinkt blau/gelb abwechselnd. Was bedeutet das?
Die interne grüne LED mach ein Doppelblinken -> 1 sec pause und wieder Doppelblinken.

Alarm- und Servicemeldungen alle grün.

Es gibt wohl ein oder 2 Geräte, die nicht an sind. Kann es daran liegen?

Welche Infos bräuchtet ihr noch?

Grüße
Thomas

**Reply by blackbasket** [reply]
Wieso sind alle Meldungen grün, wenn Geräte aus sind? Hast du die Servicemeldungen für die deaktiviert?

**Reply by thoxxxx** [reply]
Es gibt einen 4-fach Autor für die Gartenbewässerung, der aus ist, aber in der Geräteliste. Wenn man keinen Schalter betätigt, dann gibt es auch keine Servicemeldungen. Jetzt habe ich den Schalter 1 betätigt, und die Servicemeldung moppt auf, dass die Kommunikation gestört war.

Servicemeldungen sind nicht deaktiviert.

**Reply by blackbasket** [reply]
Ändert sich was, wenn du die LED für SM ausschaltest? Einstellungen -> Systemsteuerung -> Allgemein -> unten

**Reply by thoxxxx** [reply]
ja, tut es. Die LED ist jetzt blau.

**Reply by thoxxxx** [reply]
Ich habe bei allen Geräte die SM ausgeschaltet -> LED blau. Dann habe ich die SM in Einstellungen -> Systemsteuerung -> Allgemein -> unten wieder eingeschaltet -> LED blau/gelb blinkend

**Reply by blackbasket** [reply]
Dann versuche zu klären, warum du keine SM in der WebUI siehst, die LED dir diese aber trz signalisiert... da fehlt mir aber die Idee zum Test...

**Reply by thoxxxx** [reply]
Vielleicht kommt die SM von der CCU selbst. Man kann die SM der CCU bei den Geräten nicht ausschalten.

**Reply by blackbasket** [reply]
Normalerweise werden aber alle SM angezeigt... SM für einzelne Geräte kannst du unter Geräte deaktivieren...

**Reply by thoxxxx** [reply]
CCU ist das einzige Gerät mit eingeschalteten SM. Interessant, dass die LED SM anzeigt, aber die SM Anzeige grün ist.

**Comment by jens-maus** [comment]
> Normalerweise werden aber alle SM angezeigt... SM für einzelne Geräte kannst du unter Geräte deaktivieren...

Genau das wird es sein: @thoxxxx wird für einzelne Geräte die Servicemeldungen "abgeschalten" haben. Das interessiert aber den LED daemon nicht und der signalisiert auch bei abgeschalteten Servicemeldungen trotzdem diese. Ist halt so...

**Reply by blackbasket** [reply]
Ahhh, das wäre jetzt meine nächste Vermutung gewesen ;) ich wollte gerade den Gegentest anregen... Danke @jens-maus

**Reply by thoxxxx** [reply]
Hallo Jens,

ja, ok. zumindest haben wir eine Erklärung. Ich werde mein Setup demnächst etwas aufräumen. Dann sollte es keine ausgeschalteten Geräte mehr geben.

Danke euch für die Antworten!

**Reply by jens-maus** [reply]
Ergibt es keinerlei Sinn das @thoxxxx hier anscheinend bei ALLEN Geräten die Servicemeldungen abgeschalten hat. Das Abschalten der Servicemeldungen in der WebUI ist nur eine Maßnahme um die WebUI selbst davon in kenntnis zu setzen. Andere Dienste und externe Engines wie ioBroker & Co werden trotzdem diese Servicemeldungen anzeigen/anmosern. Ist also kein Bug, sondern eben eine Limitation der Umsetzung. Bei einer originalen CCU3 kann man die Servicemeldungen von geräten z.B. gar nicht abschalten.

**Reply by thoxxxx** [reply]
Ich werde mal später berichten...

**Reply by blackbasket** [reply]
> Ergibt es keinerlei Sinn das @thoxxxx hier anscheinend bei ALLEN Geräten die Servicemeldungen abgeschalten hat.

Sehe ich auch so ;)

**Reply by thoxxxx** [reply]
Ich habe erstmal die SM bei allen Geräten wieder eingeschaltet. Aber der Notifier über die LED ist aus. Das Blinken under dem Schreibtisch ging mir etwas auf die Nerven.
